example/run_djinni.sh

```diff
@@ -55,6 +55,7 @@ fi
     --java-package $java_package \
     --java-class-access-modifier "package" \
     --java-generate-interfaces true \
+    --java-implement-android-os-parcelable true \
```

Comment:
As above, I think this feature shouldn't be included in the basic example.

Reply:
I included this here, because this way, if someone is developing a new feature in djinni and that feature breaks parcelable, it will notice faster, but if you want, I can remove it.

Reply:
A good goal, though I think the test-suite covers tat, without the example having to be involved.  As-is, the example doesn't build for non-Android Java, which is one of the included build flavors.